components/sup-protocol/src/lib.rs

```diff
@@ -74,7 +74,7 @@ lazy_static! {
 
 /// Generate a new secret key used for authenticating clients to the `CtlGateway`.
 pub fn generate_secret_key(out: &mut String) {
-    let mut rng = rand::rngs::OsRng::new().unwrap();
+    let mut rng = rand::rngs::OsRng;
```

Comment:
Why did this change?

Reply:
Updated rand deprecates the old way - the new one seems better as there is no unwrap